workflow/executor/agent.go

```diff
@@ -63,7 +65,7 @@ func (ae *AgentExecutor) Agent(ctx context.Context) error {
 	defer runtimeutil.HandleCrash(runtimeutil.PanicHandlers...)
 
 	taskWorkers := env.LookupEnvIntOr(EnvAgentTaskWorkers, 16)
-	log.WithField("task_workers", taskWorkers).Info("Starting Agent s15")
+	ae.log.WithField("task_workers", taskWorkers).Info("Starting Agent s15")
```

Comment:
@simster7 why did you put `s15` here please?

Reply:
I am guessing @simster7 meant to do a `("Starting %s agent task workers", taskWorkers)`